content/en/blog/how-to-use-fission-postgresql-serverless-database/index.md

```diff
@@ -0,0 +1,170 @@
++++
+title = "How to use Fission with PostgreSQL Serverless database"
```

Comment:
This could be `How to use PostgresSQL database with Fission functions`

We cant `PostgreSQL Serverless database` as it's simple database, IMO.

Reply:
fixed